src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -1,208 +1,213 @@
 <div id="question-submission-form" class="card">
-  <div class="card-header bg-primary text-white question-header"
+  <button class="card-header bg-primary text-white question-header cursor-pointer border-0" (click)="this.model.isTabExpanded = handleClick($event);"
```

Comment:
```suggestion
  <button class="card-header bg-primary text-white question-header cursor-pointer border-0" (click)="this.model.isTabExpanded = !this.model.isTabExpanded;"
```

Should be able to simplify this without any changes in behaviour, this way we won't need to define an extra `handleClick` function.

Also, some things that can be improved:
- The dropdown and tick icons ideally should have `aria-hidden="true"`.
- Since the button expands a tab, let's bind `isTabExpanded` to the `aria-expanded` attribute.